tests/phrase.spec.test.ts

```diff
@@ -91,6 +91,23 @@ describe('::loadInContextEditorScript', () => {
             expect(document.querySelectorAll('script').length).toBe(2);
         });
     });
+
+    describe('when window is undefined for SSR', () => {
+        const { window } = global;
+
+        beforeAll(() => {
+            delete (<any>global).window;
+        });
+
+        it('runs without errors', () => {
+            const result = PhraseInContextEditorPostProcessor.loadInContextEditorScript();
+            expect(result).toBe(undefined);
```

Comment:
Should be rather something like below, right?
```suggestion
            expect(PhraseInContextEditorPostProcessor.loadInContextEditorScript).not.toThrow();
```